docs/csharp/language-reference/operators/division-operator.md

```diff
@@ -9,22 +9,36 @@ helpviewer_keywords:
 ms.assetid: d155e496-678f-4efa-bebe-2bd08da2c5af
 ---
 # / Operator (C# Reference)
-The division operator (`/`) divides its first operand by its second operand. All numeric types have predefined division operators.
-  
-## Remarks  
- User-defined types can overload the `/` operator (see [operator](../../../csharp/language-reference/keywords/operator.md)). An overload of the `/` operator implicitly overloads the [/= operator](division-assignment-operator.md).  
-  
- When you divide two integers, the result is always an integer. For example, the result of 7 / 3 is 2. This is not to be confused with floored division, as the `/` operator rounds towards zero: -7 / 3 is -2.  
-  
- To obtain a quotient as a rational number, use the `float`, `double`, or `decimal` types. There are many ways to convert between [built in numeric types](../../../csharp/language-reference/keywords/reference-tables-for-types.md).  
-  
- To determine the remainder, use the [remainder operator](../../../csharp/language-reference/operators/remainder-operator.md) `%`.  
-  
-## Example  
- [!code-csharp[csRefOperators#42](../../../csharp/language-reference/operators/codesnippet/CSharp/division-operator_1.cs)]  
-  
-## See Also
-
-- [C# Reference](../../../csharp/language-reference/index.md)  
-- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
-- [C# Operators](../../../csharp/language-reference/operators/index.md)
+
+The division operator `/` divides its first operand by its second operand. All numeric types support the division operator.
+
+## Integer division
+
+For the operands of integer types, the result of the `/` operator is of an integer type and equals the quotient of the two operands rounded towards zero:
+
+[!code-csharp-interactive[integer division](~/samples/snippets/csharp/language-reference/operators/DivisionExamples.cs#Integer)]
+
+To obtain the quotient of the two operands as a floating-point number, use the `float`, `double`, or `decimal` type:
+
+[!code-csharp-interactive[integer as floating-point division](~/samples/snippets/csharp/language-reference/operators/DivisionExamples.cs#IntegerAsFloatingPoint)]
+
+## Floating-point division
+
+For the `float`, `double`, and `decimal` types, the result of the `/` operator is the quotient of the two operands:
```

Comment:
I think it is worth mentioning that if one of the operands is `decimal`, the other can't be `float` or `double`. (There is an implicit conversion from integer to decimal, but not from float or double to decimal. You must explicitly convert a float or double argument to decimal.)

Reply:
@BillWagner thanks for review. I've addressed your feedback.